includes/functions.php

```diff
@@ -447,19 +447,26 @@ function ($matches) use ($heading_ids, $is_hidden) {
             $hid = preg_replace('/^[\s\-]+/', '', $hid); // remove dashes from start of string (e.g. for parameter)
             $heading_ids[] = $hid;
             $hidden_class = $is_hidden ? 'toc-hidden' : '';
-            return '<h' .
-                $matches[1] .
-                ' id="' .
-                $hid .
-                '" class="' .
-                $hidden_class .
-                '">' .
-                $matches[2] .
-                '<a href="#' .
-                $hid .
-                '" class="header-link scroll_to_link"><span class="fas fa-link fa-xs ms-1"></span></a></h' .
-                $matches[3] .
+            $to_return = '<h' . $matches[1] . ' id="' . $hid . '" class="' . $hidden_class . '">';
+            if(strpos($matches[2], '</i>')){
+                $to_return .= $to_return . 
+                    '<a href="#' .
+                    $hid .
+                    '" class="header-link parameter-link scroll_to_link"><span class="fas fa-link fa-xs ms-1"></span></a>' .
```

Comment:
```suggestion
                    '" class="header-link parameter-link scroll_to_link"><span class="fas fa-link fa-xs me-1"></span></a>' .
```
Is that enough margin, or do we need more (can't currently test it 😬)

Reply:
how about always showing them? somehow I have the filling it will have weird gaps otherwise?

Reply:
You mean like this?

![Skärmavbild 2023-03-29 kl  13 11 28](https://user-images.githubusercontent.com/37932118/228516410-9314b382-72c5-4583-bfe5-61a2ceee4091.png)